docs/reference/data-fetching.md

```diff
@@ -2,15 +2,28 @@
 
 Getting content and data into your React modules partials can take many forms as the sources are varied and nuanced.
 
-## HubSpot Content - Server Side
+## Server Side
 
-In an ideal world the HubSpot GraphQL integration would be the go to for getting all of your HubSpot content into React components. Currently however, GraphQL only supports querying HubDB and Custom Objects - refer to the [GraphQL](#graphql) documentation above. There are some key advantages to using the GraphQL integration with React modules.
+### GraphQL
+
+GraphQL (pro/enterprise) currently exposes the following internal HubSpot data:
+
+* HubDB
+* CRM Objects
+* Blog
+* *Coming Soon* - Knowlege Base
+
+Visit https://app.hubspot.com/graphiql/[portal-id] to explore your schema.
+
+In an ideal world the HubSpot GraphQL integration would be the go to for getting all of your HubSpot content into React components. Currently however, GraphQL only supports what is listed above - refer to the [GraphQL](./js-modules#graphql) documentation. There are some key advantages to using the GraphQL integration with React modules.
```

Comment:
Are we limiting gSSP for pro/ent as well or just the usage of GraphQL?

Reply:
Good catch.